### PR TITLE
Provide raw JSON and SDP rendering and render model JSON

### PR DESCRIPTION
**Pull request by peterbrightwell**
Checked on is-template, is-13, bcp-006-01 and ms-05-02

### CODE REVIEW

File: make-rewrites.sh [hunk 1]
```diff
@@ -55,3 +55,8 @@ fi
 
 echo Making repo rewrite rule
 echo "RewriteRule ^repo(.*) $REPO_ADDRESS\$1 [R]" >> $HTACCESS
+
+if compgen -G "*/*/examples/*.sdp" > /dev/null; then
```
Comment (garethsb) [inline make-rewrites.sh:59]: Can/should we use recursive `**` if we enable globstar and make this a bit more flexible?

Comment (peterbrightwell) [reply make-rewrites.sh:59]: Have added

**Comment by garethsb** [comment]
@peterbrightwell Can we do preorder rather than postorder depth first walk?

 image

**Review by garethsb — APPROVED** [review]
Have read through the render code changes a few times. I don't grok all of it, but it looks good, and I also checked out the rendered BCP-006-01, IS-13 and MS-05-02 indices and pages. Only reservation is the order of the control classes in their index mentioned in previous comment.

**Comment by peterbrightwell** [comment]
Thanks. That is fixed in d584957 so merging.

**Pull request merged**